plugins/CoreAdminHome/vue/src/JsTrackingCodeGenerator/JsTrackingCodeGenerator.vue

```diff
@@ -431,6 +433,24 @@ export default defineComponent({
         );
       }
 
+      if (!this.siteExcludedReferrers[idSite]) {
+        this.isLoading = true;
+
+        promises.push(
+          AjaxHelper.fetch({
+            module: 'API',
+            method: 'SitesManager.getExcludedReferrers',
+            idSite,
+            filter_limit: '-1',
+          }).then((data) => {
+            this.siteExcludedReferrers[idSite] = [];
+            (data || []).forEach((referrer: string) => {
+              this.siteExcludedReferrers[idSite].push(referrer.replace(/^https?:\/\//, ''));
```

Comment:
Would we want to also remove `http://` from excluded referrers items?

Reply:
This will remove `http` as well as `https`, that's what the `s?` is for.

Reply:
My mistake, I missed the `?` :man_facepalming: